javascript/cluster_management/src/create_single_region_cluster.js

```diff
@@ -0,0 +1,42 @@
+import { DSQLClient, CreateClusterCommand, waitUntilClusterActive } from "@aws-sdk/client-dsql";
+
+export async function createCluster(region) {
+
+    const client = new DSQLClient({ region });
+
+    try {
+        const createClusterCommand = new CreateClusterCommand({
+            deletionProtectionEnabled: true,
+            tags: {
+                Name: "javascript single region cluster"
```

Comment:
Please add the repo tag to any created clusters like the other sample do, so we can identify them and clean them up later.